cherrypy/_cplogging.py

```diff
@@ -111,14 +111,13 @@
 import logging
 import os
 import sys
-
-import six
+import uuid
 
 import cherrypy
+import six
```

Comment:
Don't move it here, as it's a third-party. It must remain between stdlib imports and the project imports.

Reply:
sorry, it was the visualstudio code import sorter, I'll not use it anymore